### PR TITLE
Disable Spanish queue

### PR DESCRIPTION
**Pull request by jasonmalinowski**
We have many test failures in this queue, so disabled for now.

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
@dotnet-bot test ci please.

File: netci.groovy [hunk 1]
```diff
@@ -80,23 +80,6 @@ commitPullList.each { isPr ->
   }
 }
 
-// Windows Spanish image
```
Comment (sharwell) [inline netci.groovy]: ❓ Can we comment this out instead, so when we enable it again we know the update is correct?

Comment (jasonmalinowski) [reply netci.groovy]: @sharwell We'll git revert the change either way. How does this help?

Comment (sharwell) [reply netci.groovy]: ➡️ Git revert should cover it, thanks!

**Pull request merged**